jsonref/__init__.py

```diff
@@ -5,6 +5,7 @@
 from urllib import parse as urlparse
 from urllib.parse import unquote
 from urllib.request import urlopen
+from . import proxytypes
```

Comment:
LazyProxy wasn't meant to be exposed to users, so I don't imagine anyone is using it directly, but should we keep the import like it was just in case anyone was doing `from jsonref import LazyProxy`?

Reply:
Oh yes, good catch! 8e94839 fixes this.